spec/requests/api/issue_tokens_spec.rb

```diff
@@ -0,0 +1,177 @@
+require 'rails_helper'
+
+describe IssueToken do
+  def api_create_issue_token(path, data, expected_status = 201)
+    api_request_issue_token(:post, path, { data: data }, expected_status)
+  end
+
+  def api_update_issue_token(path, data, expected_status = 200)
+    api_request_issue_token(:patch, path, { data: data }, expected_status)
+  end
+
+  def api_request_issue_token(method, path, params = {}, expected_status = 200)
+    send(method, "/api#{path}", params: params)
+    assert_response expected_status
+  end
+
+  let(:issue) { build(:basic_issue) }
+  let(:seed) { build(:full_natural_docket_seed, issue: issue) }
+
+  before(:each) do
+    seed.observations << [build(:observation), build(:robot_observation)]
+    issue = seed.issue
+    issue.save!
+  end
+
+  it 'show all observations from issue' do
+    issue_token = IssueToken.where(issue: issue).first
+
+    api_get("/issue_tokens/#{issue_token.token}/show_by_token")
+    expect(api_response.included.count).to eq(2)
+    expect(api_response.included.map(&:type).uniq).to eq(['observations'])
+  end
+
+  it 'responds with 410 error when token is invalid' do
+    issue_token = IssueToken.where(issue: issue).first
+    Timecop.travel 31.days.from_now
+
+    api_get("/issue_tokens/#{issue_token.token}/show_by_token", {}, 410)
+  end
+
+  it 'replies to an observation' do
```

Comment:
Qué pasa si se intenta responder más de una vez?

Reply:
Respondo lo que vi hasta el momento, si se tratara de hacer otro `api_update_issue_token`, en el segundo llamado el `IssueToken` ya fue eliminado. No encontré exactamente el motivo, pero me da error porque no puede encontrar el token ya que dejo de existir

Reply:
@fbravo-bitex te animás a agregar un spec que validate este comportamiento? es decir, intentar volver a contestar y que tire un error

Reply:
Agregué un spec para el caso en el que se consultan las observaciones de un issue las cuales todas fueron respondidas y debería retornar un listado vacío. Respecto al spec que retorne error al tratar de responder una observación con respuesta, noté que ya hay uno hecho `fails if replies to an already answered observation` (el siguiente spec)